server/pbench/bin/detect_nodename

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
So I was sifting through the existing scripts and I'm curious why don't we have this license header in them? case in point, this [index-pbench](https://github.com/distributed-system-analysis/pbench/blob/master/server/pbench/bin/index-pbench#L2) script 
Maybe I still don't fully get the concept of individual tagging of files with licenses. Because I've often come across repos that have a single LICENSE file in project root and that's the end of it..